docs/resources/rule_attack_rechecker_rewrite.md

```diff
@@ -154,5 +154,8 @@ When `type` is `absent`
 ## Attributes Reference
```

Comment:
check the sentence above

When `type` is `absent`, ...
comma is missed

why this sentence is divided into several strings?